roles/lib_utils/filter_plugins/openshift_master.py

```diff
@@ -481,10 +481,8 @@ def translate_idps(idps, api_version):
                            Dumper=AnsibleDumper))
 
     @staticmethod
-    def certificates_to_synchronize(hostvars, include_keys=True, include_ca=True):
-        ''' Return certificates to synchronize based on facts. '''
-        if not issubclass(type(hostvars), dict):
-            raise errors.AnsibleFilterError("|failed expects hostvars is a dict")
+    def certificates_to_synchronize(include_keys=True, include_ca=True):
```

Comment:
If we're removing hostvars as the first argument, we need to update all uses of this filter, otherwise it's going to fail.